kong/plugins/acme/schema.lua

```diff
@@ -53,6 +54,7 @@ local LEGACY_SCHEMA_TRANSLATIONS = {
     type = "string",
     translate_backwards = {'server_name'},
     deprecation = {
+      replaced_with = { { path = { 'server_name' } } },
```

Comment:
We have `translate_backwards` and now `replaced_with`. what is the difference here and what is this `path = ...`, there is no `path =` with ` translate_backwards`. Can we use single thing for both?

Reply:
> The plan is to use only one of them

If both `translate_backwards` and `replaced_with` can fix the bug, we'd better stick with the `translate_backwards` and avoid introducing an extra metaschema in this PR.

The refactor and introduction of `replaced_with` can be made in another PR.

Reply:
Unfortunately we cannot use only one of the since `translate_backwards` is a single just a table path and we need to have an array of table paths. The reason is that one field can be replaced by a few fields ex: `redis.timeout` and `redis.connect_timeout` / `redis.send_timeout` / `redis.read_timeout`. 
We need to have a way to check against all 3 of these.

Reply:
I've added a test case just for that. Maybe it explains it a little bit better: https://github.com/Kong/kong/pull/13565/files#diff-be157ebd3368ec7ae7c30ab60332aebbd18f627ec4fa680156e6d6cd1706f668R4233